src/main/groovy/wooga/gradle/github/base/tasks/internal/AbstractGithubTask.groovy

```diff
@@ -141,19 +143,19 @@ abstract class AbstractGithubTask<T extends AbstractGithubTask> extends Conventi
     @Optional
     @Input
     @Override
-    String getUserName() {
-        return this.userName
+    String getUsername() {
+        return this.username
```

Comment:
do we need `return` ?

Reply:
No. It is a old habit. But it doesn’t hurt. I could run a quick search and remove pattern over the code.